### ISSUE TITLE
Spot Loaders Break Quantum Entangled Singularities

### ISSUE THREAD

**Issue by Anarcraft** [AppliedEnergistics/Applied-Energistics-2#814]
Two quantum link bridges are powered via RF and are about 1k blocks away from each other.  The QNBs are on a subnet which are for the ME P2P tunnels (so I'm not limited to only 32 channels over one QNB pair) and upon server startup the link stops working.  To fix it you throw away the existing entangled singularities and create a new entangled pair.  Confirmed the bug on a test setup with only smart cable, quartz fiber, ME P2P tunnels, me chest, me terminal, QNBs, energy accepters and creative energy cell.  Removing the ME P2P tunnels prevents the issue.  Only happens on a server NOT in single player.  Also does NOT happen if both QNBs are in the same chunk.  Having a controller on either network doesn't seem to make any difference.  All chunks of the ME networks are chunkloaded.  I'm using ChickenChunks spotloaders, I guess I could try Railcraft anchors if you see any value in trying that.

I'm using a custom mod pack, let me know if you need the mods list.

Forge10.13.2.1286
appliedenergistics2-rv2-beta-7

AE addons:
EnderIO-1.7.10-2.2.7.325
ExtraCells-1.7.10-2.2.41b67
thaumicenergistics-0.8.8.4b-rv2

Java 8.25

**Comment by Deshiba** [comment]
Did you power the QNB on both sides?

**Comment by Anarcraft** [comment]
Yes, creative energy cell and power accepter on both sides.  It starts working again when I replace the entangled singularities with a new pair (the original are now broken).

**Comment by Anarcraft** [comment]
So it was 15-20 server startups that I confirmed it breaking in a row (because I was testing things) and it works with 6 startups when I removed the P2P but now it broke again.  I've added world anchors in addition to the spot loaders, I'll see what happens...

**Comment by yueh** [comment]
Are you using a security terminal?

**Comment by Anarcraft** [comment]
Oh yea, forgot to mention that.  I tested it without the security terminal and still had issues.  An update: So far I have not had the issue a single time since I have installed the world anchors (and I have now removed the spot loaders).  I put all P2P tunnels back 2 days ago and it's still good.  Due to the intermittent nature of this I want to wait a few more days before closing this but it is looking good!  I also had issues with my Big Reactor which I haven't seen since the world anchors as well so I suspect an issue with #Chicken-Bones ChickenChunks...
